app/helpers/people_helper.rb

```diff
@@ -91,5 +91,14 @@ def may_impersonate?(user, group)
       !origin_user &&
       group.people.exists?(id: user.id)
   end
-
+  
+  def link_to_address(person)
+    if !person.address.nil? && !person.zip_code.nil? && !person.town.nil?
+      link_to_address_url = 'https://nominatim.openstreetmap.org/search.php?polygon_geojson=1&viewbox='
+      link_to_address_url += '&street=' + person.address
+      link_to_address_url += '&postalcode=' + person.zip_code
+      link_to_address_url += '&city=' + person.town
+      link_to fa_icon('map-marker 2x'), link_to_address_url, target: '_blank'
```

Comment:
we have a more generic icon helper icon( ) you should use that one, so we can change more easily between different kinds of icon sets.